standalone.py

```diff
@@ -431,28 +431,35 @@ def adjust_search_path(path):
         if chapters is None:
             exit_with_error("invalid JSON: could not find field 'chapters'")
         for chapter in chapters:
+            # Extract data
```

Comment:
[nit]: Replace with `Extract data from the chapter JSON.`

Reply:
done